docs/Makefile

```diff
@@ -33,7 +33,7 @@ build: $(EXAMPLES_DIRS) sp-html
 $(BUILDDIR)/docs-downloads/examples/%.tgz: ../examples/official/%/
 	find "$<" \( -name __pycache__ -o -name \*.pyc \) -delete
 	mkdir -p $$(dirname "$@")
-	$(TAR) -czf "$@" -C "$<" .
+	$(TAR) -czf "$@" -C $(shell dirname "$<") $(shell basename "$<")
```

Comment:
Maybe `$(TAR) -czf "$@" -C "$<"/.. "$$(basename "$<")"` (i.e., (1) skip the call to `dirname` entirely, reducing the number of metacharacters flying around as a side effect, and (2) run `basename` directly in the shell instead of spawning an extra shell, passing the result through Make, and then interpolating back into the command).

Reply:
I'm not crazy about `"$<"/..` -- seems a bit ugly to do that kind of manual path manipulation (`dirname` is, at least in principle, less ad-hoc). But `$$` instead of `$(shell ...)` seems worth trying.